src/DataValues/DecimalValue.php

```diff
@@ -355,10 +355,11 @@ public function getArrayValue() {
 	 *
 	 * @deprecated since 0.8.3. Static DataValue::newFromArray constructors like this are
 	 *  underspecified (not in the DataValue interface), and misleadingly named (should be named
-	 *  newFromArrayValue). Instead, use DataValue builder callbacks in @see DataValueDeserializer.
+	 *  newFromArrayValue). Instead, use the constructor or use DataValue builder
+	 *  callbacks. (e.g. @see DataValueDeserializer)
```

Comment:
I find the "or" problematic because the possibility to pass callbacks to DataValueDeserializer is not an alternative for having constructors. The callback must construct a DataValue object. Also I find the typography (brackets outside of the sentence) weird.

`Instead, use the default constructor or introduce a new static constructor with a better name, e.g. when needed for @see DataValueDeserializer.`